src/components/Comments/DetailDomainScreenCommentItem.js

```diff
@@ -186,7 +187,7 @@ const styles = StyleSheet.create({
   btnReplyText: {
     fontFamily: fonts.inter[400],
     fontSize: 13,
-    color: '#C4C4C4',
+    color: COLORS.gray1,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Again, replacing hard-coded color values with dynamic ones from a theme object is a good practice. It improves modularity and consistency.
```diff
-    color: '#C4C4C4',
+    color: COLORS.gray1,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->